lib/sheldon/memory.rb

```diff
@@ -0,0 +1,16 @@
+require 'yaml/store'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.